atlasdb-timelock-server/src/main/java/com/palantir/atlasdb/timelock/config/ClusterConfiguration.java

```diff
@@ -25,16 +25,14 @@
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import com.google.common.base.Preconditions;
 
-import io.atomix.catalyst.transport.Address;
-
 @JsonSerialize(as = ImmutableClusterConfiguration.class)
 @JsonDeserialize(as = ImmutableClusterConfiguration.class)
 @Value.Immutable
 public abstract class ClusterConfiguration {
-    public abstract Address localServer();
+    public abstract String localServer();
```

Comment:
This is being mapped to Address, so the string must be of the form ``host:port``, this will throw if this happens to be ``https://host:port/``. We should add the validation before ``.map(Address::new)``. Perhaps this will not be valuable for the Atomix impl, but should be kept in mind when creating a new impl.